adapter/prometheus2/prometheus.go

```diff
@@ -94,11 +97,12 @@ func (b *builder) clearState() {
 	b.metrics = make(map[string]*cinfo)
 }
 
-func (b *builder) ConfigureMetricHandler(map[string]*metric.Type) error { return nil }
-
-func (b *builder) Build(c adapter.Config, env adapter.Env) (adapter.Handler, error) {
+func (b *builder) SetMetricTypes(map[string]*metric.Type) {}
+func (b *builder) SetAdapterConfig(cnfg adapter.Config)   { b.cnfg = cnfg.(*config.Params) }
```

Comment:
Please abbreviate "configuration" to either "config" or "cfg", that's what we do in the rest of the codebase.

Reply:
Done